### PR TITLE
Instant sealing engine

### PR DESCRIPTION
**Pull request by keorn**
To be used for development.

### CODE REVIEW

File: ethcore/res/sealing_engine.json [hunk 1]
```diff
@@ -0,0 +1,33 @@
+{
+	"name": "Priv",
```
Comment (arkpar) [inline ethcore/res/sealing_engine.json]: This gets displayed on startup. A proper name would be nice.

File: ethcore/src/engines/sealing_engine.rs [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+use engines::Engine;
+use spec::CommonParams;
+use evm::Schedule;
+use env_info::EnvInfo;
+use block::ExecutedBlock;
+use common::Bytes;
+use account_provider::AccountProvider;
+
+/// An engine which does not provide any consensus mechanism, just seals blocks internally.
+pub struct SealingEngine {
```
Comment (arkpar) [inline ethcore/src/engines/sealing_engine.rs]: How is different from `NullEngine`? Also the name is too generic. Engine == SealingEngine

File: ethcore/res/sealing_engine.json (hunk 1 above)
Comment (keorn) [inline ethcore/res/sealing_engine.json]: Do you mean if someone uses this test json file?

File: ethcore/src/engines/sealing_engine.rs (hunk 2 above)
Comment (keorn) [inline ethcore/src/engines/sealing_engine.rs]: `NullEngine` does not seal (always `None`). I agree, what do you think about `InstantEngine`?

File: ethcore/res/sealing_engine.json (hunk 1 above)
Comment (arkpar) [inline ethcore/res/sealing_engine.json]: People also use these file as reference

File: ethcore/src/engines/sealing_engine.rs (hunk 2 above)
Comment (arkpar) [inline ethcore/src/engines/sealing_engine.rs]: `InstantEngine` or `InstantSeal` is ok.
If it creates a seal it should also verify one.

Comment (keorn) [inline ethcore/src/engines/sealing_engine.rs]: It creates an empty seal and accepts every seal. Ok, I will change to `InstantSeal`.

**Comment by gavofyork** [comment]
tests failing.

**Pull request merged**